duneanaobj/StandardRecord/StandardRecord.h

```diff
@@ -40,12 +50,30 @@ namespace caf
     int reco_q;
     float Elep_reco;
     float theta_reco;
+    int reco_lepton_pdg;
 
     float RecoLepEnNue;
     float RecoHadEnNue;
     float RecoLepEnNumu;
     float RecoHadEnNumu;
 
+    double pileup_energy;
+
+    SRNDLAr  ndlar;
+    SRTMS  ndtms;
+
+    std::vector<caf::SRNDTrackMatch> ndtrkmatches;
+
+    // gas TPC info
+    int nFSP;
+    std::vector<int> pdg;
+    std::vector<float> ptrue;
+    std::vector<float> trkLen;
+    std::vector<float> trkLenPerp;
+    std::vector<float> partEvReco;
+    int gastpc_pi_pl_mult;
+    int gastpc_pi_min_mult;
```

Comment:
Should these not get their own class like `SRGAr`?

Reply:
For sure they should.  I copied these names directly from the old ND_CAFMaker output (so that we at least don't *lose* info when we switch to this style of CAF) and figured that somebody who knew what they were doing would come along and deal with the GAr stuff later.  But I guess I can nest them here too, and the GAr folks can slot more stuff in as needed

Reply:
now stuffed into a `SRGAr`, see 4eb84e9d